Xamarin.Forms.Platform.iOS/Platform.cs

```diff
@@ -478,6 +479,30 @@ static void PresentPopUp(UIWindow window, UIAlertController alert, ActionSheetAr
 
 		internal class DefaultRenderer : VisualElementRenderer<VisualElement>
 		{
+			public override UIView HitTest(CGPoint point, UIEvent uievent)
+			{
+				// UIview hit testing ignores objects which have an alpha of less than 0.01 
+				// (see https://developer.apple.com/reference/uikit/uiview/1622469-hittest)
+				// To prevent layouts with low opacity from being implicitly input transparent, 
+				// we need to temporarily bump their alpha value during the actual hit testing,
+				// then restore it. If the opacity is high enough or user interaction is disabled, 
+				// we don't have to worry about it.
+
+				nfloat old = Alpha;
+				if (UserInteractionEnabled && old <= 0.01)
+				{
+					Alpha = (nfloat)0.011;
```

Comment:
No. The next CALayer update won't happen until after this method is done. Since we're setting the Alpha back to its original value before exiting the method, nothing visibly happens. And since we're only touching a CALayer property, the UIView's Draw method is never triggered.